src/app/pages/research/projects-section/projects-section.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from 'react';
+import LinkOrNot from '../link-or-not';
+import cn from 'classnames';
+import './projects-section.css';
+
+const projectColors = ['blue', 'green', 'yellow', 'red', 'orange'];
+
+export default function ProjectsSection({data: {projectsHeader, projects, ...other}}) {
+    return (
+        <section className="projects-section">
+            <div className="content">
+                <h1>{projectsHeader}</h1>
+                <div className="project-cards">
+                    {
+                        projects.map((project, i) =>
+                            <div className={cn('card', projectColors[i])} key={project.title}>
```

Comment:
will projects length always be less than projectColors?  otherwise you'll be reading an invalid index, which probably is fine since classnames will ignore undefined, but then the div would not have a color class.

Reply:
Using modula length.